test-plugins/org.yakindu.sct.simulation.core.sexec.test/test-gen/org/yakindu/sct/simulation/core/sexec/test/StatechartEntryExitActions.java

```diff
@@ -34,22 +34,8 @@ public void entryActionsAreExecutedOnEnteringStatechart() throws Exception {
 		assertTrue(getInteger("x") == 5l);
 		assertTrue(getInteger("y") == 3l);
 	}
-	public void entryActionsAreExecutedOnEnteringStatechartOnlyIfGuardIsTrue() throws Exception {
```

Comment:
Did someone remove these test cases from the sctunit test class? These test cases are fine as they make guard checking explicit.

Reply:
some tests failed because the expectations were wrong, can you recheck this please?